src/components/Wizard/CreateVmWizard/providers/vmwareActions.js

```diff
@@ -181,3 +187,23 @@ export const onVCenterVmSelectedConnected = async (
     );
   }
 };
+
+/**
+ * Provides mapping from VMWare GuesId to common-templates operating system.
+ *
+ * https://code.vmware.com/docs/4206/vsphere-web-services-api-reference#/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+ *
+ * The VMWARE_TO_KUBEVIRT_OS_CONFIG_MAP_NAME object is usually created by the web-ui-operator and can be missing.
+ *
+ * @param operatingSystems - see getTemplateOperatingSystems() in selectors.js
+ * @param guestId - VMWare's operating system identifier
+ */
+export const getVmwareToKubevirtOS = async (operatingSystems, guestId, k8sGet) => {
+  const vmwareToKubevirtOsConfigMap = await k8sGet(
```

Comment:
I am missing some kind of error handling here

Reply:
good catch, added. Presence of the object is not required.

Reply:
last nitpick: you could also log the error..just in case :) otherwise LGTM

Reply:
done